PORTAL/jobs/_pyperformance.py

```diff
@@ -1049,7 +1052,7 @@ def parse(
                 except StopIteration:
                     continue
                 if not headerdiv.startswith('+='):
-                #if headerdiv != header.div:
+                    #if headerdiv != header.div:
```

Comment:
We could probably drop this line.  If we keep it, the indent should stay as it was.